costmap_2d/src/costmap_2d.cpp

```diff
@@ -219,6 +224,23 @@ bool Costmap2D::worldToMap(double wx, double wy, unsigned int& mx, unsigned int&
   return false;
 }
 
+bool Costmap2D::worldToMapContinuous(double wx, double wy, float& mx, float& my) const
+{
+  if (wx < origin_x_ || wy < origin_y_)
+  {
+    return false;
+  }
+
+  mx = static_cast<float>((wx - origin_x_) / resolution_) + 0.5f;
+  my = static_cast<float>((wy - origin_y_) / resolution_) + 0.5f;
+
+  if (mx < size_x_ && my < size_y_)
+  {
+    return true;
+  }
+  return false;
```

Comment:
```suggestion
  return mx < size_x_ && my < size_y_;
```
You litteraly wrote if true return true if false return false. XD